packages/react/src/components/Header/Header.stories.tsx

```diff
@@ -1,11 +1,154 @@
-/* eslint-disable react/jsx-key */
 import React from 'react';
 
+import myLogo from '../../../../../assets/img/digdir-logo.svg';
+import { Link } from '../Link/Link';
+
 import { Header } from './Header';
+import classes from './HeaderStories.module.css';
+
+const AllChildren = [
+  <Header.Left key={0}>
+    <a href='.'>
+      <img
+        src={myLogo}
+        alt='SVG logo'
+      />
+    </a>
+  </Header.Left>,
+  <Header.Middle key={1}>
+    <nav>
```

Comment:
The HTML inside the header regions for the story seem to repeat themselves a lot (DRY). Maybe try to make the code more reusable.